docs/install.rst

```diff
@@ -67,23 +106,32 @@ You may also need to install the `Gulp <http://gulpjs.com/>`_ cli globally::
 .. _run-python-tests:
 
 Run the tests
--------------
-
-.. Important::
-
-    We're working on fixing this, but for now you need the localization files for the tests to pass.
-    See the `Localization`_ section below for instructions on checking those out.
+=============
 
 Now that we have everything installed, let's make sure all of our tests pass.
 This will be important during development so that you can easily know when
-you've broken something with a change. You should still have your virtualenv
+you've broken something with a change.
+
+Docker
+------
+
+We manage our local docker environment with docker-compose and Make. All you need to do here is run::
+
+    $ make test
```

Comment:
Is it possible to test a single app this way?

Reply:
Pls add this to the docs if it's not there already.